programs/clearing_house/src/controller/liquidation.rs

```diff
@@ -373,6 +373,7 @@ pub fn liquidate_perp(
             oracle_price,
             base_asset_amount: user_position_delta.base_asset_amount,
             quote_asset_amount: user_position_delta.quote_asset_amount,
+            lp_shares: user_lp_shares,
```

Comment:
weren't we going to move `burn_lp_shares` right under the cancel market orders block?

Reply:
incorpd